docs/getting_started.rst

```diff
@@ -1,8 +1,76 @@
+===============
 Getting Started
 ===============
 
-This page includes example notebooks intended for new users of `xrtpy`.
+XRTpy is a Python package being developed for the analysis of observations made by the X-Ray Telescope (XRT)
+on the board Hinode spacecraft. This page is intended for new users of `xrtpy`. For more background information about XRT please refer to the `SolarSoft XRT Analysis Guide`_.
+
+
+XRTpy Objects:
+**************
+XRTpy currently offers `~xrtpy.response.channel.Channel`, *Effective Area*, and
+*Temperature Response* classes. These classes allow users
+to explore properties of the XRT. Visit our Example page for detail example guides on how to use the XRTpy classes.
+
+
+Channel
+-------
+Channel is an instrument configuration class that explores properties of the XRT. Channel offers a detailed review of instruments for a chosen
+filter channel including the Charge-Coupled Device (CCD), Entrance Filter, Focus-Filter(s), Geometry, and Mirror(s).
+
+
+Effective Area
+--------------
+XRTpy produces the effective areas for a set of XRT filter channels paired with thicknesses of the CCD contamination layer.
+Refer to the `SolarSoft XRT Analysis Guide`_ for more information about the instrumental spectral responses.
+
+
+Temperature Response
+--------------------
+XRTpy produces the temperature response for each XRT filter channel, assuming a spectral emission model, refer to `Narukage et al. 2011`_ and `Narukage et al. 2014`_.
```

Comment:
```suggestion
XRTpy produces the temperature response for each XRT filter channel, assuming a spectral emission model, refer to :cite:t:`Narukage:2011` and :cite:t:`Narukage:2014`.
```
THis would require adding these entries to `docs/bibliography.bib` too